client/src/style/MainMenu.css

```diff
@@ -0,0 +1,40 @@
+.parent {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        padding: 10%;
+        background-color: dimgray;
+}
+
+.parent-box {
+        background-color: #333;
+        border-radius: 12px;
+        color: #eee;
+        min-height: 200px;
+        padding: 150px;
+        width: 300px;
+
+        display: flex;
+        flex-flow: column;
```

Comment:
If you only want to specify the flex direction just use the `flex-direction` property